### PR TITLE
inverse laplace and mellin transforms missing 2*pi*I

### PR DESCRIPTION
**Pull request by cbm755**
#### Brief description of what is fixed or changed

Fixes bug #14946, missing factor of `2*pi*I` in some inverse integral transforms.

#### Release Notes

* integrals
  * Fix missing factor of `2*pi*I` in inverse Laplace and Mellin transforms.

### CODE REVIEW

**Comment by jksuom** [comment]
Should the factor be added to the docstrings as well?

**Comment by cbm755** [comment]
Yes, I thought I did so with the latex changes.  Or do you mean I should doctest the results of  `as_integral`?

**Comment by jksuom** [comment]
I was thinking of the doctstrings of `inverse_mellin_transform` and `inverse_laplace_transform` (.. math: formulas).

File: sympy/integrals/transforms.py [hunk 1]
```diff
@@ -819,7 +819,7 @@ def inverse_mellin_transform(F, s, x, strip, **hints):
 
     This can be defined as
 
-    .. math:: f(x) = \int_{c - i\infty}^{c + i\infty} x^{-s} F(s) \mathrm{d}s,
+    .. math:: f(x) = \frac{1}{2\pi i} \int_{c - i\infty}^{c + i\infty} x^{-s} F(s) \mathrm{d}s,
```
Comment (cbm755) [inline sympy/integrals/transforms.py]: @jksuom here?

File: sympy/integrals/transforms.py [hunk 2]
```diff
@@ -1234,14 +1234,14 @@ def _compute_transform(self, F, s, t, **hints):
...
-        return Integral(exp(s*t)*F, (s, c - I*oo, c + I*oo))
+        return Integral(exp(s*t)*F, (s, c - I*oo, c + I*oo))/(2*S.Pi*S.ImaginaryUnit)
 
 
 def inverse_laplace_transform(F, s, t, plane=None, **hints):
     r"""
     Compute the inverse Laplace transform of `F(s)`, defined as
 
-    .. math :: f(t) = \int_{c-i\infty}^{c+i\infty} e^{st} F(s) \mathrm{d}s,
+    .. math :: f(t) = \frac{1}{2\pi i} \int_{c-i\infty}^{c+i\infty} e^{st} F(s) \mathrm{d}s,
```
Comment (cbm755) [inline sympy/integrals/transforms.py]: and here, somewhere else I missed?

File: sympy/integrals/transforms.py (hunk 1 above)
Comment (jksuom) [reply sympy/integrals/transforms.py]: Yes, of course. My mistake. I did not recognize that as part of docstring.

**Pull request merged**